src/add/operator/timestamp.ts

```diff
@@ -0,0 +1,10 @@
+import {Observable} from '../../Observable';
+import {timestamp, TimeStampSignature} from '../../operator/timestamp';
+
+Observable.prototype.timestamp = timestamp;

```

Comment:
This is following RxJS 4 naming, https://github.com/Reactive-Extensions/RxJS/blob/master/doc/api/core/operators/timestamp.md

`timestamp` doesn't follow camelcase, `s` is not capital. Was there specific reason for this?